### DISCUSSION TITLE
Passing motor as a parameter using block programming

### DISCUSSION THREAD

**Discussion by fmorton** [pybricks/support#1814] [General]
![IMG_4832](https://github.com/user-attachments/assets/74a85c98-fb09-42d6-abac-7eb297689ca3)

In this example, showing the python code being generated from block programming, I would like to pass the "note_motor_1" from line 9 as the first parameter in the "move_mallet" method/function. Easy enough to do in python, but is there a way to do this using block programming?

Thanks.

**Comment by laurensvalk** [comment]
Yes, you can find the created objects on the variables tab. See https://github.com/orgs/pybricks/discussions/1344#discussioncomment-7915280

**Comment by fmorton** [comment]
That is awesome. Did not notice it. Thanks.
